tools/mo/requirements.txt

```diff
@@ -1,6 +1,6 @@
 numpy>=1.16.6
 networkx~=2.5; python_version <= "3.6"
```

Comment:
```suggestion
```

Reply:
3.6 is not officially supported anymore, maybe we need to keep this line as legacy?